test/DOSProxy_test.js

```diff
@@ -0,0 +1,219 @@
+<<<<<<< c96e7712e9d4f0e6787398dbeb327bb8d7d1ecfc
```

Comment:
Basically the test for DOSProxy and DOSOnChainSDK contracts are also not necessary, as majority functionalities are in library.  You could remove these 2.